run

```diff
@@ -1,5 +1,38 @@
 #!/bin/bash
 
+# DKIM config
+dkimConfig()
+{
+    postconf -e milter_protocol=2
+    postconf -e milter_default_action=accept
+    postconf -e smtpd_milters=inet:localhost:12301
+
+    rm -f /etc/opendkim/KeyTable
+    rm -f /etc/opendkim/SigningTable
+
+    echo "DNS records:"
+    for d in $OPENDKIM_DOMAINS ; do
+      domain=$(echo "$d"| cut -f1 -d '=')
+      selector=$(expr match "$d" '.*\=\(.*\)')
+      if [ -z "$selector" ] ; then
+        selector="mail"
+      fi
+
+      DIR="/etc/opendkim/keys/$domain"
+      privateFile=$DIR/$selector.private
+      txtFile=$DIR/$selector.txt
```

Comment:
This will be backwards compatible right?

Reply:
My tests say yes :)

Reply:
👍